pkgs/sdk/vm/vm_wrapper.go

```diff
@@ -0,0 +1,118 @@
+package vm
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/gnolang/gno/pkgs/errors"
+	gno "github.com/gnolang/gno/pkgs/gnolang"
+	"github.com/gnolang/gno/pkgs/log"
+	"github.com/gnolang/gno/stdlibs"
+)
+
+type Wrapper struct {
```

Comment:
Is there a more specific name we can use? 
An EvalContext?

Reply:
I'm not sure this structure is really making the program easier to understand.

Reply:
I do agree that we want some kind of refactoring but I think it would be better to revert the Wrapper stuff here and to make a separate PR later with perhaps a different refactor.

Reply:
yes, absolutely, I'm splitting it out to another PR, https://github.com/gnolang/gno/pull/718.

Reply:
yes, absolutely, I'm splitting it out to another PR, https://github.com/gnolang/gno/pull/718.